runners/spark/src/main/java/org/apache/beam/runners/spark/aggregators/NamedAggregators.java

```diff
@@ -126,7 +126,7 @@ public NamedAggregators merge(NamedAggregators other) {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     for (Map.Entry<String, State<?, ?, ?>> e : mNamedAggregators.entrySet()) {
-      sb.append(e.getKey()).append(": ").append(e.getValue().render());
+      sb.append(e.getKey()).append(": ").append(e.getValue().render()).append("\n");
```

Comment:
This is there to make the `String` representation of the aggregators (and also metrics) more human readable. This is in addition to the fact that this `String` representation also shows up in `Spark`'s UI allowing you to view the accumulated values in each stage. However, `\n` didn't work out well here, so will probably change it to a space character.